Source/Helpers/Extensions/PHCachingImageManager+Extensions.swift

```diff
@@ -21,15 +21,29 @@ extension PHCachingImageManager {
     }
     
     func fetchImage(for asset: PHAsset,
-					cropRect: CGRect,
-					targetSize: CGSize,
 					callback: @escaping (UIImage, [String: Any]) -> Void) {
         let options = photoImageRequestOptions()
     
         // Fetch Highiest quality image possible.
         requestImageData(for: asset, options: options) { data, _, _, _ in
             if let data = data, let image = UIImage(data: data)?.resetOrientation() {
-            
+
+                let exifs = self.metadataForImageData(data: data)
```

Comment:
Is there a possibility for a retain cycle here? I see the `callback` is `@escaping`.  Can we capture `self` weakly?